deploy/nomadTasks.nix

```diff
@@ -169,6 +169,7 @@ rec
     env = {
       HOST = "0.0.0.0";
       PORT = "\${NOMAD_PORT_marlowe_load}";
+      QUERY_PORT = "\${NOMAD_PORT_contract_query}";
```

Comment:
@jhbertra , will this addition break the Nomad infrastructure or require an addition to it?

Reply:
I think it should be fine as long as you add the `"contract_query"` to the `config.ports` below.

Reply:
I think it should be fine as long as you add the `"contract_query"` to the `config.ports` below.

Reply:
Fixed